sympy/core/relational.py

```diff
@@ -1075,9 +1081,375 @@ def _n2(a, b):
-def _eval_is_ge(lhs, rhs):
-    return None
+def _eval_is_ge(lhs, rhs):  # noqa:F811
+    from sympy.simplify.radsimp import fraction
+    hit = 0
+    for i in (lhs, rhs):
+        if not i.is_extended_real:
+            ilhs = _invpow(lhs)
+            irhs = _invpow(rhs)
+            if ilhs.is_extended_real and irhs.is_extended_real:
+                return is_ge(irhs, ilhs)
+            return
+        if i.is_Mul or i.is_Pow or i.is_Add:
+            hit += 1
+    if not hit:
+        return
+    ix = False  # keep track of whether L and R were exchanged
+    L, R = lhs, rhs
+    if L.is_Rational:
+        L, R = R, L
+        ix = ~ix
+    if L.is_number and R.is_number:
+        L, R = L - R, S.Zero
+    if not R:
+        if not L:
+            return True  # lhs == rhs already handled before getting here
+        if L.is_extended_negative:
+            return True if ix else False
+        elif L.is_extended_positive:
+            return False if ix else True
+        elif L.is_extended_nonnegative:
+            return None if ix else True
+        elif L.is_extended_nonpositive:
+            return True if ix else None
+        if L.is_Add:
+            a, b = L.as_two_terms()
+            if a.is_Rational:
+                L = b
+                R = -a
+            else:
+                L = a
+                R = -b
+    # simplify wrt Rationals
+    m, l, b = _aslinear(L)
+    l = _trivial(l)
+    if l is None:
+        #assert None,(11)
+        return
+    mr, r, br = _aslinear(R)
+    r = _trivial(r)
+    if r is None:
+        #assert None,(10)
+        return
+    # make processing canonical
+    if l.is_Number and not r.is_Number:
+        l, r = r, l
+        m, mr = mr, m
+        b, br = br, b
+        ix = ~ix
+    if SYMPY_DEBUG:
+        print(lhs, rhs, '-trivial->', l, r)
+        if pause: input('press return to continue')
+
+    # work really hard to identify a power for comparison
+    if l.is_Pow and r == l.base:
+        r = Pow(r, 1, evaluate=False)
+    elif r.is_Pow and l == r.base:
+        l = Pow(l, 1, evaluate=False)
+    if l.is_Pow and r.is_Pow:
+        if l == r:  # m*x + b ~~ mr*x + br
+            if m == mr:
+                L, R = b - br, S.Zero
+                if ix:  # probably impossible at this point
+                    L, R = R, L
+                return L >= R
+            L = l
+            R = br - b
+            den = m - mr
+            if den < 0:
+                ix = ~ix
+            R /= den
+        elif b == br:
+            if l.base == r.base:
+                if is_ge(S.Zero, r):
+                    ix = ~ix
+                elif not is_ge(r, S.Zero):
+                    return  # hopefully impossible after passing entrance test
+                L = Pow(l.base, l.exp - r.exp, evaluate=False)
+                R = mr/m
+                if m < 0:
+                    ix = ~ix
+            elif m == mr and l.exp == r.exp and (
+                    l.exp.is_integer or
```

Comment:
These conditions are too strict and are keeping the `_base_like_pow` lines unused.